docs/components/best-practices/modeling/choosing-the-resource-binding-type.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Choosing the resource binding type
+tags:
+  - BPMN
+description: "Understand the differences between 'latest' and 'deployment' binding for linked resources"
+---
+
+For the following elements in a BPMN process, you can choose the binding type for the linked target resource:
+
+- [call activities](/docs/components/modeler/bpmn/call-activities/call-activities.md#defining-the-called-process)
+- [business rule tasks](/docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md#defining-a-called-decision) (if the DMN decision implementation is used)
+- [user tasks](/docs/components/modeler/bpmn/user-tasks/user-tasks.md#user-task-forms) (if a Camunda Form is linked)
+
+The binding type determines the version of the target resource that will be used at runtime.
+For example, in case of a call activity, it would be the version of the called process to be instantiated.
+
+The following binding types are currently supported in Camunda 8:
+
+- **`latest`** will resolve to the **latest deployed version** of the target resource (at the point in time the process element is activated).
+  Be aware that this can lead to unexpected behavior if you deploy a new version of the target resource without ensuring backwards compatibility with every deployed process that depends on it.
+  Therefore, using `latest` might not be suited for production environments that require stability and a predictable behavior.
+
+- **`deployment`** will resolve to the particular version of the target resource that was **deployed together** with the currently running version of the process in the **same deployment**.
```

Comment:
A user might wonder: "how do I deploy multiple resources in the same deployment"
Answer: Use a process application's deployment UI or use the Zeebe deployment API with multiple files (needs to be revised for technical accuracy)
 
Links to these docs pages would be helpful
https://docs.camunda.io/docs/apis-tools/zeebe-api/gateway-service/#deployresource-rpc
https://docs.camunda.io/docs/next/components/modeler/web-modeler/process-applications/

Reply:
Thanks, good suggestion! 👍🏻